screens/Menu/NearbyInterestDetails.js

```diff
@@ -1,33 +1,151 @@
+
+            <SafeAreaView style={styles.buttonContainer}>
+
+                <View style={styles.backArrowContainer}>
+                    <Button transparent style={styles.backArrow} onPress={goBack}>
+                        <Icon name="md-arrow-round-back" style={styles.icon}></Icon>
+                    </Button>
+                </View>
+
+                {name.length > 19 &&
+                    <Text style={styles.mainLabel}>{name.substring(0, 19) + "..."}</Text>
+                }
+                {name.length <= 19 &&
+                    <Text style={styles.mainLabel}>{name}</Text>
+                }
+
+                <Text style={styles.totalReviewLabel}>{reviews ? reviews + " Reviews" : "N/A"}</Text>
+
+                <Button transparent style={styles.ratingContainer}>
+                    <View style={styles.ratingNumberContainer}>
+                        <Text style={styles.reviewLabel}>{rating ? rating : "N/A"}</Text>
+                    </View>
+                    <View style={styles.separator}></View>
+                    <View style={styles.buttonTextContainer}>
+                        <View style={styles.ratingStarsContainer}>
+                            <StarRating disabled={false} maxStars={5} rating={rating} fullStarColor={"#fcba03"} starSize={20} />
+                        </View>
+                    </View>
+                </Button>
+
+                <Button transparent style={styles.hourButton}>
+                    <View style={styles.iconContainer}>
+                        <Icon type="Feather" name="clock" style={styles.iconBtn}></Icon>
+                    </View>
+                    <View style={styles.separator}></View>
+                    <View style={styles.buttonTextContainer}>
+                        <Text style={styles.btnLabel}>{opening ? opening : "N/A"}</Text>
+                    </View>
+                </Button>
+
+                <Button transparent style={styles.mapButton}>
+                    <View style={styles.iconContainer}>
+                        <Icon type="Feather" name="map-pin" style={styles.iconBtn}></Icon>
+                    </View>
+                    <View style={styles.separator}></View>
+                    <View style={styles.buttonTextContainer}>
+                        <Text style={styles.btnLabel}>{address ? address : "N/A"}</Text>
+                    </View>
+                </Button>
+
+                <Button transparent style={styles.phoneButton}>
+                    <View style={styles.iconContainer}>
+                        <Icon type="Feather" name="phone" style={styles.iconBtn}></Icon>
+                    </View>
+                    <SafeAreaView style={styles.separator}></SafeAreaView>
+                    <View style={styles.buttonTextContainer}>
+                        <Text style={styles.btnLabel}>{phone ? phone : "N/A"}</Text>
```

Comment:
Maybe `Coming Soon` instead of N/A.
Same for web

Reply:
Good idea!